src/mapml-viewer.js

```diff
@@ -341,6 +342,14 @@ export class MapViewer extends HTMLElement {
 
     this._layerControl = M.layerControl(null,{"collapsed": true, mapEl: this}).addTo(this._map);
 
+    let scaleValue = M.options.announceScale;
+
+    if (typeof scaleValue === "string") {
+      scaleValue = JSON.parse(scaleValue);
+    }
+
+    this._scaleBar = M.scaleBar(scaleValue).addTo(this._map);
```

Comment:
I think this should be conditional on it not existing yet, as I believe this is how the controls work now.  i.e. if (!this._scaleBar) this._scaleBar = ...

Reply:
Maybe? Also perhaps there's a method on L.ScaleBar to allow you to switch measurement system dynamically?